src/test/eo/org/eolang/txt/text-tests.eo

```diff
@@ -445,3 +445,17 @@
         text " 漢Hello, dude  "
       0
     $.equal-to "漢"
+
+[] > temp-test
+  QQ.io.stdout > @
+    QQ.txt.sprintf
+      "char : %d\n"
+      as-int.
+        bytes-1-to-8
+          as-bytes.
+            "a"
+
+  [b] > bytes-1-to-8
```

Comment:
I mean intead of this one

Reply:
@levBagryansky it's garbage in the code, I forgot to delete it, I'll fix it now.